src/scene/assetmanager.cpp

```diff
@@ -144,52 +144,55 @@ void AssetManager::deinitialize() {
     _toBeDeleted.clear();
 }
 
-void AssetManager::update() {
-    ZoneScoped;
-
-    // Flag to keep track of when to emit synchronization event
-    const bool isLoadingAssets = !_toBeInitialized.empty();
-
-    // Delete all the assets that have been marked for deletion in the previous frame
-    {
-        ZoneScopedN("Deleting assets");
-
-        _toBeDeleted.clear();
-    }
-
-    // Initialize all assets that have been loaded and synchronized but that not yet
-    // initialized
-    for (auto it = _toBeInitialized.cbegin(); it != _toBeInitialized.cend(); it++) {
-        ZoneScopedN("Initializing queued assets");
-        Asset* a = *it;
+void AssetManager::runRemoveQueue() {
+    ZoneScopedN("Removing queued assets");
```

Comment:
```suggestion
    ZoneScoped;
```

The `N` version was previously needed since it was embedded in a different function.  Now that it is its own function we can use the `ZoneScoped` which takes the name from the function directly